src/auth/serializer/src/operations.js

```diff
@@ -352,7 +414,8 @@ let comment_options = new Serializer(
     allow_votes: bool,
     allow_curation_rewards: bool,
     extensions: set(static_variant([
-        comment_payout_beneficiaries
+        comment_payout_beneficiaries,
+        allowed_vote_assets
```

Comment:
Will this break pre-HF serialization for `comment_options`, as well as the other 2 which were modified?

Reply:
No. Static Variants were intended to be expanded this way. Old serializations continue to work as is. Any new static variant will either fail deserialization if sent to an old node or be rejected steemd via a hardfork check.